README.md

```diff
@@ -74,6 +74,32 @@ Follow the given instructions for Login into the app.
    ````
    Fill up the things it asked to and then Login into the app.
 
+## Environment Variables
+
+1. zulip_id : You can go [Zulip](https://anitab-org.zulipchat.com) and click on your name from the user list on the right. On the URL, you will see a number after user. That is your Zulip user ID.
+
+2. SENDGRID_API_KEY : Follow the given steps to create a Sendgrid API key -
+
+	- Go to [Sendgrid's website](https://app.sendgrid.com/guide)
```

Comment:
this list could be numbered list instead of bullet pointed list, since these steps follow a certain order.